scan/lib/scan/test_command_generator.rb

```diff
@@ -37,6 +37,7 @@ def options
         options << "-resultBundlePath '#{result_bundle_path}'" if config[:result_bundle]
         options << "-enableCodeCoverage YES" if config[:code_coverage]
         options << "-enableAddressSanitizer YES" if config[:address_sanitizer]
```

Comment:
not an xcode expert, but would there be a use case where wants to use the CL to override the project's config, in which case we should not also pass the value NO when the config of `address_sanitizer` or `thread_sanitizer ` explicitly pass the value `false` ?